backend/server/src/http/mod.rs

```diff
@@ -187,7 +193,11 @@ async fn handle(req: Request<Body>, ctx: Arc<Context>) -> Result<Response> {
 
         // The actual GraphQL API.
         (&Method::GET, "/graphql") | (&Method::POST, "/graphql") => {
-            juniper_hyper::graphql(ctx.api_root.clone(), ctx.api_context.clone(), req).await?
+            juniper_hyper::graphql(
+                ctx.api_root.clone(),
+                Arc::new(api::Context::new(ctx.db.get().await?).await?),
+                req
+            ).await?
```

Comment:
Trailing comma after `req`. Or pull out the second argument as `let db_conn = ...;` and write the `graphql` call in one line.

Reply:
Moved it back. :)